sw/device/lib/flash_ctrl.h

```diff
@@ -21,6 +21,14 @@
  */
 typedef enum bank_index { FLASH_BANK_0 = 0, FLASH_BANK_1 = 1 } bank_index_t;
 
+/**
+ * Flash partitions
+ */
+typedef enum partition_type {
+  DATA_PARTITION = 0,
+  INFO_PARTITION = 1
```

Comment:
minor nit: use kCamelCase for enum literals to match coding style

Reply:
ah yes thanks. let me fix that.

Reply:
updated.